src/Haskell/Ide/Engine/Plugin/Pragmas.hs

```diff
@@ -71,13 +81,16 @@ codeActionProvider plId docId _ (J.CodeActionContext (J.List diags) _monly) = do
 
 -- ---------------------------------------------------------------------
 
+-- | Find all Pragmas are an infix of the search term.
 findPragma :: T.Text -> [T.Text]
 findPragma str = concatMap check possiblePragmas
```

Comment:
this could be just
`findPragma str = filter (\p -> T.isInfixOf p str) possiblePragmas` right?

Reply:
I think so.  But that change should be in a separate PR, this one is documentation only